integration-tests/tests/import-cohort-data.js

```diff
@@ -31,6 +31,7 @@ const testImportCohortDataFn = _.flow(
   await findElement(page, clickable({ textContains: 'an existing workspace' }))
   await waitForNoSpinners(page)
   await click(page, clickable({ textContains: 'an existing workspace' }))
+  await delay(3000) // need to wait for responses to finish loading
```

Comment:
Instead of having a delay, you should wait for the page to be in the state you expect. Delays are flaky (they may not be long enough), and they unnecessarily add length to tests (they may be too long).

Reply:
I'm not sure what page the state should be in. We already `await waitForNoSpinners(page)` earlier, but the workspaces take time to load so they are not available when the spinners complete.